src/app/main/header/header.component.ts

```diff
@@ -100,6 +100,8 @@ export class HeaderComponent implements OnInit, OnDestroy{
     }
 
     doSearch(search: SearchParams) {
+
+        console.log(search);
```

Comment:
can this be removed?

Reply:
Ups, yes. I will remove it now and merge it asap. So, I can create new release afterwards.

Thanks @mdelez for reviewing it

Reply:
Done in faa233d

Reply:
Damn...the tests failed. But why?